server.py

```diff
@@ -14,7 +14,28 @@
 from db import Code, LoginCode, PasswordReset, PendingEmail, User
 import util
 
-config = util.loadYaml('config.yaml')
+# The reason for including this up here instead of down below is so we can
+# non-intrusively change some constants before they are used.
+
+# temp vars:
+config_path = 'config.yaml'
+
+if __name__ == '__main__':
+	from argparse import ArgumentParser
+
+	# Initialize startup arguments
+	parser = ArgumentParser(description='FunboxUsers Account server')
+	parser.add_argument(
+		'--config-path', metavar='config_path', type=str,
+		help='Path to the config yaml. DEFAULT='+'./config.yaml',
+		default='config.yaml'
```

Comment:
We should define a `DEFAULT_CONFIG_PATH` instead of repeating `config.yaml` several times here